lang/java/compiler/pom.xml

```diff
@@ -232,6 +232,10 @@
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.jetbrains</groupId>
+      <artifactId>annotations</artifactId>
+    </dependency>
```

Comment:
I only see the dependency being used in the test code (which is perfect IMHO). Can we please reduce the scope to test?